docs/creating/config.md

```diff
@@ -78,12 +78,12 @@ It has a very high cost associated with its use, so it will only appear when abs
 ### `config.build.mode`
 
 In the `build` section we have `mode`.
-A model can run in `plan`, `operate`, or `spores` mode.
-In `plan` mode, capacities are determined by the model, whereas in `operate` mode, capacities are fixed and the system is operated with a receding horizon control algorithm.
-In `spores` mode, the model is first run in `plan` mode, then run `N` number of times to find alternative system configurations with similar monetary cost, but maximally different choice of technology capacity and location (node).
+A model can run in `base`, `operate`, or `spores` mode.
+In `base` mode, capacities are determined by the model, whereas in `operate` mode, capacities are fixed and the system is operated with a receding horizon control algorithm.
```

Comment:
This could be confusing if someone updates `base` to their own math as `base` mode is then whatever they have defined and may not include capacities being determined by the model...